Multisig/UI/Safe Management/Create Safe/Setup Safe/CreateSafeFormUIModel.swift

```diff
@@ -34,7 +34,7 @@ class CreateSafeFormUIModel {
     var sectionHeaders: [CreateSafeFormSectionHeader] = []
     var state: CreateSafeFormUIState = .initial
     var futureSafeAddress: Address?
-    var userSelectedPaymentMethod = Transaction.PaymentMethod.signerAccount
+    var userSelectedPaymentMethod: Transaction.PaymentMethod? = nil
```

Comment:
why we need this to be nil?

Reply:
Because the user hasn't decided yet. So no override of an other default